<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QWidget" name="widgetWeight" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="FontRadioButton" name="radioButtonArial">
         <property name="text">
          <string>Arial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FontRadioButton" name="radioButtonTimes">
         <property name="text">
          <string>Times New Roman</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FontRadioButton" name="radioButtonComic">
         <property name="text">
          <string>Comic Sans MS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="TextLineEdit" name="lineEditSize"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelFamily">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelFox">
      <property name="text">
       <string>The quick brown fox likes CSE 335</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Family</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="SizeSlider" name="sliderEditSize">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="SizeComboBox" name="comboBoxSize">
      <item>
       <property name="text">
        <string>small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>big</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
   <zorder>labelFamily</zorder>
   <zorder>lineEditSize</zorder>
   <zorder>label</zorder>
   <zorder>widgetWeight</zorder>
   <zorder>label_3</zorder>
   <zorder>labelFox</zorder>
   <zorder>label_2</zorder>
   <zorder>sliderEditSize</zorder>
   <zorder>comboBoxSize</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">textlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SizeSlider</class>
   <extends>QSlider</extends>
   <header location="global">sizeslider.h</header>
  </customwidget>
  <customwidget>
   <class>FontRadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">fontradiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>SizeComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">sizecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
